neqo-transport/src/connection/tests/zerortt.rs

```diff
@@ -129,22 +125,8 @@ fn zero_rtt_send_reject() {
     client
         .enable_resumption(now(), token)
         .expect("should set token");
-    let mut server = Connection::new_server(
```

Comment:
This is a different test case.  I would prefer that you add a new test case for your new thing.